qute-quickstart/src/test/java/org/acme/qute/NativeHelloResourceIT.java

```diff
@@ -0,0 +1,9 @@
+package org.acme.qute;
+
+import io.quarkus.test.junit.NativeImageTest;
+
+@NativeImageTest
+public class NativeHelloResourceIT extends HelloResourceTest {
```

Comment:
Shouldn't you have a native test for `ItemsResourceTest` too?

Reply:
Oops, that was an oversight. But I'm not sure it's worth it...